polkadot/node/core/pvf/tests/it/main.rs

```diff
@@ -21,12 +21,20 @@ use polkadot_node_core_pvf::{
 	start, Config, InvalidCandidate, Metrics, PrepareJobKind, PvfPrepData, ValidationError,
 	ValidationHost, JOB_TIMEOUT_WALL_CLOCK_FACTOR,
 };
-use polkadot_parachain_primitives::primitives::{BlockData, ValidationParams, ValidationResult};
+use polkadot_parachain_primitives::primitives::{
+	BlockData as GenericBlockData, ValidationParams, ValidationResult,
+};
 use polkadot_primitives::ExecutorParams;
 
-#[cfg(feature = "ci-only-tests")]
+#[cfg(any(feature = "ci-only-tests", feature = "tracking-allocator"))]
 use polkadot_primitives::ExecutorParam;
 
+#[cfg(feature = "tracking-allocator")]
+use ::adder::{hash_state, BlockData, HeadData};
```

Comment:
Can we have the new tests in `adder.rs`, so we don't need these inputs?

Reply:
Imports are no more needed after I switched to your `precheck_pvf`, so I've left tests where they are. It seems to me that the current separation is rather logical; `main.rs` contains general PVF host tests, and `adder.rs` is about chain production tests.

Reply:
> main.rs contains general PVF host tests, and adder.rs is about chain production tests.

Would you mind adding docstrings for the two files? I didn't understand the logic for the separation and there are no comments.